apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/Chat.tsx

```diff
@@ -232,7 +224,7 @@ export default function Chat({
         />
         <ChatTextArea
           placeholder='Enter followup message...'
-          disabled={responseStream !== undefined}
+          disabled={responseStream !== undefined || !!error}
```

Comment:
I explicitly left it enabled when erroring. We want users to retry if things fail.